setup.py

```diff
@@ -1,17 +1,37 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
-import os
+import sys, os
 import numpy as np
 from Cython.Build import cythonize
 
 moab_env_var = 'MOAB_PATH'
+moab_root = None
+moab_default = '/usr/'
 
-try:
-    moab_root = os.environ[moab_env_var]
-except KeyError:
-    raise EnvironmentError('MOAB_PATH not found in environment.')
+#look for command line argument
+for arg in sys.argv:
+    if "--moab-path" in arg:
+        moab_root = arg.split("=")[-1]
```

Comment:
this should probably be `arg.split("=", 1)` in case the path itself contains an equals sign.